app/policies/like_policy.rb

```diff
@@ -0,0 +1,16 @@
+class LikePolicy < ApplicationPolicy
+  attr_reader :user, :like
+
+  def initialize(user, like)
+    @user = user
+    @like = like
+  end
+
+  def create?
+    user
+  end
+
+  def destroy?
+    user == like.fan
```

Comment:
consider comparing their id's as direct object comparisons can be buggy

Reply:
agreed